app/sidekiq/event_bus_gateway/letter_ready_email_job.rb

```diff
@@ -9,12 +9,18 @@ class LetterReadyEmailJob
 
     sidekiq_options retry: 0
     NOTIFY_SETTINGS = Settings.vanotify.services.benefits_management_tools
+    HOSTNAME_MAPPING = {
+      'dev-api.va.gov' => 'dev.va.gov',
+      'staging-api.va.gov' => 'staging.va.gov',
+      'api.va.gov' => 'va.gov'
+    }.freeze
 
     def perform(participant_id, template_id)
       notify_client.send_email(
         recipient_identifier: { id_value: participant_id, id_type: 'PID' },
         template_id:,
-        personalisation: { host: Settings.hostname, first_name: get_first_name_from_participant_id(participant_id) }
+        personalisation: { host: HOSTNAME_MAPPING[Settings.hostname] || Settings.hostname,
```

Comment:
```suggestion
        personalisation: { host: HOSTNAME_MAPPING[Settings.hostname] || 'va.gov',
```

I would default to a working frontend URL rather than an API url.

Reply:
The downside of this change is it'll make the local-env links point to va.gov rather than the local env.